File: app/src/main/java/com/example/myapplication/model/Medico.java
package com.example.myapplication.model;

import java.util.Arrays;

public class Medico {
    private int id;
    private String nome;
    private String especialidade;
    private String exames[];
    private String observação;
    private int gestante;
    private int idData;
    /**
     * Recebe um nome
     * */
    public Medico(String nome, String especialidade, String[] exames, String observação, int gestante, int idData) {
        this.nome = nome;
        this.especialidade = especialidade;
        this.exames = exames;
        this.observação = observação;
        this.gestante = gestante;
        this.idData = idData;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEspecialidade() {
        return especialidade;
    }

    public void setEspecialidade(String especialidade) {
        this.especialidade = especialidade;
    }

    public String getExames() {
        String strExame = Arrays.toString(exames);
        strExame = strExame.replace("]", "");
        return  strExame.replace("[", "");
    }

    public void setExames(String[] exames) {
        this.exames = exames;
    }

    public String getObservação() {
        return observação;
    }

    public void setObservação(String observação) {
        this.observação = observação;
    }

    public int getGestante() {
        return gestante;
    }

    public void setGestante(int gestante) {
        this.gestante = gestante;
    }

    public int getIdData() {
        return idData;
    }

    public void setIdData(int idData) {
        this.idData = idData;
    }

    @Override
    public String toString() {
        return "Medico{" +
                "id=" + id +
                ", nome='" + nome + '\'' +
                ", especialidade='" + especialidade + '\'' +
                ", exames=" + Arrays.toString(exames) +
                ", observação='" + observação + '\'' +
                ", gestante=" + gestante +
                ", idData=" + idData +
                '}';
    }
}

File: SERVIDOR/Model/Medico.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Medico
 *
 * @author luan
 */


class Medico {
    private $id;
    private $nome;
    private $especialidade;
    private $exames;
    private $observacao;
    private $gestante;
    private $idRegistro;
    private $idData;
    
    function __construct($nome, $especialidade, $exames, $observacao, $gestante, $idRegistro, $idData) {
        $this->nome = $nome;
        $this->especialidade = $especialidade;
        $this->exames = $exames;
        $this->observacao = $observacao;
        $this->gestante = $gestante;
        $this->idRegistro = $idRegistro;
        $this->idData = $idData;
    }
    
    
    function getIdRegistro() {
        return $this->idRegistro;
    }

    function setIdRegistro($idRegistro) {
        $this->idRegistro = $idRegistro;
    }

        
    function getId() {
        return $this->id;
    }

    function getNome() {
        return $this->nome;
    }

    function getEspecialidade() {
        return $this->especialidade;
    }

    function getExames() {
        return $this->exames;
    }

    function getObservacao() {
        return $this->observacao;
    }

    function getGestante() {
        return $this->gestante;
    }

    function getIdData() {
        return $this->idData;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setNome($nome) {
        $this->nome = $nome;
    }

    function setEspecialidade($especialidade) {
        $this->especialidade = $especialidade;
    }

    function setExames($exames) {
        $this->exames = $exames;
    }

    function setObservacao($observacao) {
        $this->$observacao = $observacao;
    }

    function setGestante($gestante) {
        $this->gestante = $gestante;
    }

    function setIdData($idData) {
        $this->idData = $idData;
    }

    function toString(){
        return $this->nome +"-"+ $this->especialidade +"-"+$this->exames +"-"+$this->observacao+ "-"+ $this->gestante + "-";
    }


}
